server/service/integration_mdm_dep_test.go

```diff
@@ -1928,154 +1930,79 @@ func (s *integrationMDMTestSuite) TestDeprecatedDefaultAppleBMTeam() {
 	require.Equal(t, tm.Name, acResp.MDM.DeprecatedAppleBMDefaultTeam)
 }
 
-func (s *integrationMDMTestSuite) TestSetupExperienceScript() {
```

Comment:
Double-checking, it seems like the rest of the changes here are just moving the tests to a new file. Anything new in particular that I should review?